ocamldoc/odoc_module.ml

```diff
@@ -13,13 +13,12 @@
 (*                                                                        *)
 (**************************************************************************)
 
-module String = Misc.Stdlib.String
-
 (** Representation and manipulation of modules and module types. *)
 
+module String = Misc.Stdlib.String
+
 module Name = Odoc_name
 
-(** To keep the order of elements in a module. *)
```

Comment:
I think it is better to either keep all documentation comments for this group of mutually recursive types.